source/assets/js/playground.ts

```diff
@@ -155,7 +154,8 @@ function setupPlayground(): void {
     }
   }
 
-  function goToSelection(selection: PlaygroundState['selection']): void {
+  /** Updates state, applies selection in editor, and focus editor. */
```

Comment:
It's fine for documentation comments to be sentence fragments that starts with verbs, but otherwise please use proper grammar. If you refer to parameters or other code objects as proper nouns, use `{@link ...}` to refer to them. Also, try to focus on what the semantics of the function are rather than just describing its implementation in prose. For example: `Highlights {@link selection} in the code editor.`

Reply:
Updated in ee7c1d5e11c255736b0c1fda68bfcfcb8d5f8672